lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -183,8 +183,8 @@ function dosomething_northstar_build_ns_user($user, $form_state) {
   }
   foreach ($address as $ns_key => $drupal_key) {
     $field = $user->field_address[LANGUAGE_NONE][0];
-    if (isset($field[$drupal_key]['value'])) {
-      $ns_user[$ns_key] = $field[$drupal_key]['value'];
+    if (isset($field[$drupal_key])) {
+      $ns_user[$ns_key] = $field[$drupal_key];
```

Comment:
I think this was incorrect because when I `dpm(($field[$drupal_key]['value']));` it returns `B`. When I 
`dpm($field[$drupal_key]);` it returns `BR` (the country code I used in the header). Also, when I check if it reaches inside the `if (isset($field[$drupal_key]['value'])) block, it doesn't. 

Might have been assumed that it was the same format as the `foreach ($optional as $ns_key => $drupal_key)` above - checked that and correctly returns the information!